### PR TITLE
Update footer.md

### PR DESCRIPTION
**Pull request by claudiaifrim**
Fix import error for Grid.

### CODE REVIEW

File: docs/voltohandson/footer.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ Since we need the Logo component in the Footer, we import it from Volto as we di
 ```jsx
 import { Logo } from '@plone/volto/components';
 ```
+We will also need the Grid component from semantic ui:
```
Comment (ksuess) [inline docs/voltohandson/footer.md:21]:
```suggestion
We also need the `Grid` component from Semantic UI
```
For consistency: presence

**Review by ksuess — CHANGES_REQUESTED** [review]
Good catch!

**Review by stevepiercy — APPROVED** [review]
Thank you!

**Comment by stevepiercy** [comment]
Oops I meant to provide the same suggestions as @ksuess. With those changes I approve.

**Review by stevepiercy — APPROVED** [review]
I made the correction, and approve. Ready to merge.

**Pull request merged**